### PR TITLE
Add ColorPicker component.

### PR DESCRIPTION
**Pull request by furybean**
Please makes sure these boxes are checked before submitting your PR, thank you!

* [ ] Make sure you follow Element's contributing guide ([中文](https://github.com/ElemeFE/element/blob/master/.github/CONTRIBUTING.zh-CN.md) | [English](https://github.com/ElemeFE/element/blob/master/.github/CONTRIBUTING.en-US.md)).
* [ ] Make sure you are merging your commits to `dev` branch.
* [ ] Add some descriptions and refer relative issues for you PR.

### CODE REVIEW

**Comment by baiyaaaaa** [comment]
恭喜发财

**Comment by reverland** [comment]
但是，默认要引用这个吗

**Comment by QingWei-Li** [comment]
@reverland 可以按需引或者自己 build 一份嘛

**Comment by furybean** [comment]
没有 want review 的 tag，先加个 want merge。i18n 还没改完，测试还少几个，其他都 okay 了。

**Comment by Leopoldthecoder** [comment]
@furybean 测试挂了

**Comment by furybean** [comment]
@Leopoldthecoder 测试好了，各位代码看起来。

**Comment by Leopoldthecoder** [comment]
@baiyaaaaa @QingWei-Li 一起来看一波，合了就能发 1.2 了

**Comment by furybean** [comment]
@Leopoldthecoder 样式还没人看呢，就发 1.2 了？

**Comment by Leopoldthecoder** [comment]
@furybean 发之前肯定会给他们看的

File: examples/docs/en-US/color-picker.md [hunk 1]
```diff
@@ -0,0 +1,100 @@
...
+    color: #8492a6;
+    line-height: 44px;
+  }
+</style>
+
+## ColorPicker
+
+ColorPicker is a color picker component that is used to solve the need to select a color in certain scenes.
+
+### 选择颜色
```
Comment (QingWei-Li) [inline examples/docs/en-US/color-picker.md]: 中文

File: examples/docs/en-US/color-picker.md [hunk 2]
```diff
@@ -0,0 +1,100 @@
...
+        color1: '#ff0',
+        color2: null
+      }
+    }
+  };
+</script>
+```
+:::
+
+### 选择颜色和透明度
```
Comment (QingWei-Li) [inline examples/docs/en-US/color-picker.md]: 中文2

File: examples/docs/en-US/color-picker.md [hunk 3]
```diff
@@ -0,0 +1,100 @@
...
+        color4: null
+      }
+    }
+  };
+</script>
+```
+:::
+
+### Attributes
+| 参数      | 说明    | 类型      | 可选值       | 默认值   |
```
Comment (QingWei-Li) [inline examples/docs/en-US/color-picker.md]: 中文34567

File: examples/docs/en-US/color-picker.md (hunk 1 above)
Comment (furybean) [inline examples/docs/en-US/color-picker.md]: 🐶

File: examples/nav.config.json [hunk 4]
```diff
@@ -219,6 +219,10 @@
             {
               "path": "/collapse",
               "title": "Collapse 折叠面板"
+            },
+            {
+              "path": "/color-picker",
+              "title": "ColorPicker"
```
Comment (Leopoldthecoder) [inline examples/nav.config.json]: "title": "ColorPicker 颜色选择器"

Comment (Leopoldthecoder) [inline examples/nav.config.json]: 合掉了，我来改吧。

**Pull request merged**